install/github/github.go

```diff
@@ -0,0 +1,330 @@
+// Copyright 2022 OpenSSF Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
interesting. I had never seen this SPDX metadata :-)

Reply:
Hehe, you can get this with the following command:

```shell
addlicense -l apache -c 'OpenSSF Authors' -v -s *
```

`-s` is the flag that adds the SPDX identifier :)